easybuild/easyconfigs/f/fontconfig/fontconfig-2.13.1-GCCcore-9.3.0.eb

```diff
@@ -0,0 +1,39 @@
+easyblock = 'ConfigureMake'
+
+name = 'fontconfig'
+version = '2.13.1'
```

Comment:
There's a version of fontconfig 2.13.92 out there ...

Reply:
Doh, totally missed that when I was looking...
Updating....